applications/sasquatch/values-idfprod.yaml

```diff
@@ -0,0 +1,177 @@
+strimzi-kafka:
+  kafka:
+    externalListener:
+      tls:
+        enabled: true
+      bootstrap:
+        loadBalancerIP: "34.55.132.0"
+        host: sasquatch-kafka-bootstrap.lsst.cloud
+
+      brokers:
+        - broker: 3
+          loadBalancerIP: "34.122.37.250"
+          host: sasquatch-kafka-3.lsst.cloud
+        - broker: 4
+          loadBalancerIP: "34.72.131.177"
+          host: sasquatch-kafka-4.lsst.cloud
+        - broker: 5
+          loadBalancerIP: "34.72.103.157"
+          host: sasquatch-kafka-5.lsst.cloud
+  # mirrormaker2:
+  #   enabled: true
+  #   source:
+  #     bootstrapServer: sasquatch-dev-kafka-bootstrap.lsst.cloud:9094
+  #     topicsPattern: "registry-schemas, lsst.sal.*, lsst.dm.*"
+  #   resources:
+  #     requests:
+  #       cpu: 2
+  #       memory: 4Gi
+  #     limits:
+  #       cpu: 4
+  #       memory: 8Gi
+  users:
+  #   tsSalKafka:
+  #     enabled: true
+  #   camera:
+  #     enabled: true
+  #   replicator:
+  #     enabled: true
+    kafdrop:
+      enabled: true
+    telegraf:
+      enabled: true
+  #   kafkaConnectManager:
+  #     enabled: true
+  kraft:
+    enabled: true
+  kafkaController:
+    enabled: true
+    resources:
+      requests:
+        # memory: 16Gi
+        memory: 8Gi
+        # cpu: "2"
+        cpu: "1"
```

Comment:
For this and other resource limits and requests, I don't think this needs to be as big as the other installations given that it's only going to handle app metrics.

Reply:
Sounds good, we can start with that.